base/linalg/dense.jl

```diff
@@ -211,6 +211,25 @@ end
     kron(A, B)
 
 Kronecker tensor product of two vectors or two matrices.
+
+```jldoctest
+julia> A = [1 2; 3 4]
+2×2 Array{Int64,2}:
+ 1  2
+ 3  4
+
+julia> B = [1 1; 1 1]
```

Comment:
Apologies, all ones was my suggestion (to make what `kron` does easy to grasp). Can you think of an equally clear but more exciting example? Best!